### ISSUE TITLE
508-defect-2 [COGNITION]: Content MUST be legible to users on all viewports and up to 400x zoom

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#17789]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a user using a mobile device, a tablet, or magnification, I want to be able to read my listed issues so that I can select them.

## Details

Several spacing problems exist with the issue checkboxes including:
- On smaller mobile devices, the spacing becomes distorted.
- When an error exists, the text on the tablet and magnified (250x) experience wraps over itself.

## Acceptance Criteria

- [ ] Spacing is consistent across device sizes and magnified states
- [ ] Text does not wrap on top of other text

## Steps to Recreate

1. Reach the eligible issues page `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/eligible-issues` in test case 2.
2. Continue without making a selection.
3. Zoom into the page at 250x or set the width of the page to around 600-800px using devtools.
-->

## Solution

It seems like many of these problems are stemming from the various padding/margin classes. It should be a relatively straightforward fix.

## Screenshots or Trace Logs

<img width="1606" alt="Screen Shot 2020-12-21 at 1 31 40 PM" src="https://user-images.githubusercontent.com/14154792/102810814-63961b00-4392-11eb-9dad-bfb12e2bb378.png">

https://user-images.githubusercontent.com/14154792/102810819-6729a200-4392-11eb-88ad-0d15e15eb005.mov

<img width="611" alt="Screen Shot 2020-12-21 at 1 36 23 PM" src="https://user-images.githubusercontent.com/14154792/102810828-698bfc00-4392-11eb-80fa-e4b5571821ff.png">

**Comment by joshkimux** [comment]
https://user-images.githubusercontent.com/14154792/102923656-1a130200-445e-11eb-9084-261e4126f80e.mov

**Comment by Mottie** [comment]
Updates to the issue cards will be available in staging shortly

**Comment by joshkimux** [comment]
Confirmed this has been fixed. Fantastic stuff from @Mottie !
